### PR TITLE
Add tokens for Azure

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Required to get going with azure

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ env:
         # Add the GH_TRAVIS_TOKEN environment variable for the "cf-blacksmithy" Travis user
         - secure: "FLt1rYmNUTuL56kzhd+reLFk9OHQaJD79e2hItmnhXp2gGKalNkxasMhs4UboTnwzrDe9ymu+Oem47u5PvI6r/wenS8nT/o1y5klh2APi8gqjTFk2zWeDl5lhpH30k8KS15EMQBpU6jfGAER025GeNVaO3HT0ogoavf06106CuqNkrsJHCZvW2WTPXe1sDonmUNWbG3jifdLVy4CdzVQThRgzG8i/xh8IZd8205B+G2jivZhmWbCcS5JdA3tKzeVr0zztlyi5V5MlQm4hQ3UP8WlmEQ0m3+S0Ab7cXn1AhKDw5YvVaPNpyuDVe+qXMZEiCn2zVnCUvh5XJdvCfN0twxhM2Qv/q+KpIjKYR78TjlXuEc1YCITrTbHKUDW2G5i0AvnGQC7aBMNKHUvMFEso82I9+WMcm1SVydl4fo2B9wLb1Bi5FTiUO8IUsfdmuZZYqvRaLa1n1AhJ76v9hBMWePmxs0Uwh0g5tHO1hIe45X5XEQKy3hRXCo4k9M28esWloUzbTisczafcY2R5ofo58rfI+aqVzArSCF1KF+UgBkL2HJPQQsl+SBh2Lv3jC6vBWiQczwwRTHZKIsnwKj8VWlDpJHHEDpIK32PniYWpDmxA7+dKV7DHHctiYaxDkVEBlE11TXGik8HbTQuC3mL7ANSTtWaaDv2UWr9RTnXSzI="
 
+        # Add the AZURE_TOKEN environment variable used to manage pipelines on azure devops
```
Comment (isuruf) [inline .travis.yml]: Can you mention the user for the AZURE_TOKEN here?

Comment (mariusvniekerk) [reply .travis.yml]: done

**Comment by mariusvniekerk** [comment]
@isuruf @scopatz please review

**Comment by scopatz** [comment]
LGTM, @isuruf - any thoughts?

**Pull request merged**